dex-load/src/main/scala/com/wavesplatform/dex/load/utils/package.scala

```diff
@@ -0,0 +1,111 @@
+package com.wavesplatform.dex.load
+
+import java.io.{File, PrintWriter}
+import java.nio.file.Files
+
+import com.softwaremill.sttp.{HttpURLConnectionBackend, MonadError => _, _}
+import com.typesafe.config.ConfigFactory
+import com.wavesplatform.dex.domain.bytes.codec.Base58
+import com.wavesplatform.wavesj.json.WavesJsonMapper
+import com.wavesplatform.wavesj.matcher.Order
+import com.wavesplatform.wavesj.matcher.Order.Type
+import com.wavesplatform.wavesj.{ApiJson, AssetPair, PrivateKeyAccount, Transactions}
+import pureconfig._
+import pureconfig.generic.auto._
+
+import scala.io.Source
+import scala.util.Random
+
+package object utils {
+
+
+  val settings = ConfigSource
+    .fromConfig(ConfigFactory.parseResources(scala.util.Properties.envOrElse("CONF", "devnet.conf")).getConfig("waves.dex.load"))
+    .load[Settings]
+    .right
+    .get
+
+  val services         = new Services(settings)
+  val issuer           = PrivateKeyAccount.fromSeed(settings.richAccount, 0, settings.chainId.charAt(0).toByte)
+  implicit val backend = HttpURLConnectionBackend()
+
+  def mkOrderHistoryHeaders(account: PrivateKeyAccount, timestamp: Long = System.currentTimeMillis): Map[String, String] = Map(
+    "Timestamp" -> timestamp.toString,
+    "Signature" -> services.matcher.getOrderHistorySignature(account, timestamp)
+  )
+
+  def getOrderBook(account: PrivateKeyAccount): String = {
+    sttp
+      .get(uri"${settings.hosts.matcher}/matcher/orderbook/${Base58.encode(account.getPublicKey)}")
+      .headers(mkOrderHistoryHeaders(account))
+      .send()
+      .body
+      .right
+      .get
+  }
```

Comment:
Why dont return JsObject instead of string? I bet you will parse it wherever later

Reply:
ok, returning JsValue here is a good idea, fixed